src/Components/Form/Form.js

```diff
@@ -9,7 +9,6 @@ const Form = ({filterValue, sortByLanguage}) => {
         <select value={filterValue} onChange={(e) => sortByLanguage(e.target.value)} id="languageFilter">
           <option value=""></option>
           <option value="C">C</option>
-          <option value="C+">C+</option>
```

Comment:
Nice catch

Reply:
Yea I realized C+ isn't an actual language when looking for a C+ icon :)